socrates/lib/site/views/index.jade

```diff
@@ -60,61 +60,66 @@ block content
               h5
                 +extlink('http://www.codefreeze.fi/', 'CodeFreeze')
                 br
-                small 3 - 4 Feb 2015, Kiilopää, Finland
+                small 11 - 12 Jan 2016 , Kiilopää, Finland
             .col-sm-6
               h5
-                +extlink('http://socrates-conference.es/', 'SoCraTes Canaries 2015')
+                +extlink('http://www.socrates-conference.es/', 'SoCraTes Canaries')
                 br
-                small 26 Feb - 1 Mar 2015, Tenerife
+                small 10 - 13 Mar 2016, Gran Canaria
             .col-sm-6
               h5
-                +extlink('http://2015.itakeunconf.com/', 'I T.A.K.E. Unconference')
+                +extlink('http://itakeunconf.com/', 'I T.A.K.E. Unconference')
                 br
-                small 28 - 29 May 2015, Bucharest
+                small 19 - 20 May 2016, Bucharest
             .col-sm-6
               h5
-                +extlink('http://socratesuk.org/', 'SoCraTes UK 2015')
+                +extlink('http://socratesuk.org/', 'SoCraTes UK')
                 br
-                small 4 - 7 June 2015, Farncombe Estate
+                small 2 - 5 June 2016, Farncombe Estate
             .col-sm-6
               h5
                 +extlink('http://socrates-day.ch/', 'SoCraTes Day Switzerland')
                 br
-                small 19 June 2015, Zurich
+                // TODO update when they have new dates: small 19 June 2015, Zurich
             .col-sm-6
               h5
-                +extlink('http://socrates-fr.github.io', 'SoCraTes France 2015')
+                +extlink('http://socrates-fr.github.io', 'SoCraTes France')
                 br
-                small 24 - 27 Sept 2015, Rochegude
+                // TODO update when they have new dates: small 24 - 27 Sept 2015, Rochegude
             .col-sm-6
               h5
-                +extlink('http://socratesbe.org', 'SoCraTes BE 2015')
+                +extlink('http://socratesbe.org', 'SoCraTes BE')
                 br
-                small 22 - 25 Oct 2015, Nieuwpoort
+                // TODO update when they have new dates: small 22 - 25 Oct 2015, Nieuwpoort
 
 
 
     .logocolumn.col-sm-4.col-md-3.col-lg-4.col-md-offset-1.col-lg-offset-0
       .sidebar
         .page-header
-          h2 Our Sponsors
-        each pair, index in sponsors
-          .row
-            +sponsorCol(pair.first)
-            if (pair.second)
+          h2 Sponsors needed
```

Comment:
Hmm... not sure about that. I would rather have a "clean slate" at the start of the sponsoring work. I think showing last year's sponsors makes it less clear that we really need sponsors (because we already display so many).